### DISCUSSION TITLE
[Question] Display the log of flatpak-managed-install.service

### DISCUSSION THREAD

**Discussion by Nirlvy** [gmodena/nix-flatpak#84] [Q&A] [answered]
Is it possible to display the log of flatpak-managed-install.service to the output of nixos-rebuild? Or maybe there's already a way to do it？

**Comment by gmodena** [comment, marked as answer]
hey @Nirlvy,

I looked into this feature in #7. Last time I investigated, I couldn't find a decent solution that worked for both NixOS modules and HM installs. It's still on my radar, but I haven't spent much time on it yet. Ideas are very welcome :).

My workaround for the lack of progress output—though you may already be doing this—is to query the systemd unit at activation with:

```
watch systemctl status flatpak-managed-install.service
```
(Add `--user` to `systemctl` if you installed nix-flatpak as an HM module.)

**Comment by jonocodes** [comment]
Perhaps there is a way to just add a single item to the log that says something like "flatpacks installing" or something - without granular feedback. I mention this because it often seems like the build hangs, but really flatpack is cranking and I dont know thats whats happening.
